javascript/javascript1/week4/Voice-Assistant.js

```diff
@@ -0,0 +1,86 @@
+let vname = '';
```

Comment:
A very silly change it seems, still pointing out as it can turn into a bigger mistake later on. Try not to use this type of naming like vname, rather userName, even something more descriptive like voiceAssistantName :)

Reply:
okay .. thanks .. fixed now